lib/src/builder/parsing/members/props_and_state_mixins.dart

```diff
@@ -80,7 +80,7 @@ abstract class BoilerplatePropsOrStateMixin extends BoilerplateTypedMapMember
         final node = this.node;
         if (node is MixinDeclaration) {
           final isOnBaseClass = node.onClause?.superclassConstraints
-                  ?.any((type) => type.nameWithoutPrefix == propsOrStateBaseClassString) ??
+                  .any((type) => type.nameWithoutPrefix == propsOrStateBaseClassString) ??
```

Comment:
If `onClause` is possibly null - how is it ok to remove this?

Reply:
In null safety, chained null-awares now [short-circuit](https://dart.dev/null-safety/understanding-null-safety#smarter-null-aware-methods)!